index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href='https://fonts.googleapis.com/css?family=Sofia' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Flash sale</title>
+</head>
+<body>
+    <div class="container">
+        <header class="header">
+            <h1 id="title">
+                FLASH SALE STORE
+            </h1>
+            <p id="description">
+                Quality is our priority
+            </p>
+        </header>
+        <form id="survey-form">
+            <div class="form-group">
+                <label id="name-label" for="name">First Name </label>
+                <input type="text" name="name" id="name" class="formcontrol" placeholder="Enter your name" required>
+                <hr class="div-line">
+            
+            </div>
+            <div class="form-group">
+                <label id="lname-label" for="lname">Last Name </label>
+                <input type="text" name="lname" id="lname" class="formcontrol" placeholder="Enter your last name" required>
+                <hr class="div-line">
+            </div>
+            <div class="form-group">
+                <label id="phone-label" for="phone" >Phone </label>
+                <input pattern="" type="tel" name="phone" id="phone" class="formcontrol" placeholder="Enter your phone number" required>
```

Comment:
This is an input for phone number it means only phone number should be allowed but i can not fix it.